apps/ledger-live-mobile/src/screens/SendFunds/07-SendBroadcastError.tsx

```diff
@@ -0,0 +1,132 @@
+import { CompositeScreenProps } from "@react-navigation/native";
+import React, { useCallback } from "react";
+import { useTranslation } from "react-i18next";
+import { Linking, ScrollView } from "react-native";
+import { SafeAreaView, useSafeAreaInsets } from "react-native-safe-area-context";
+import { useSelector } from "react-redux";
+import styled from "styled-components/native";
+import { Download, Support } from "@ledgerhq/icons-ui/native";
+import { getAccountCurrency } from "@ledgerhq/live-common/account/helpers";
+import { Button, Flex, Text } from "@ledgerhq/native-ui";
+
+import { track, TrackScreen } from "~/analytics";
+import type { SendFundsNavigatorStackParamList } from "~/components/RootNavigator/types/SendFundsNavigator";
+import type { BaseNavigatorStackParamList } from "~/components/RootNavigator/types/BaseNavigator";
+import type { StackNavigatorProps } from "~/components/RootNavigator/types/helpers";
+import GenericErrorView from "~/components/GenericErrorView";
+import Card from "~/components/Card";
+import useExportLogs from "~/components/useExportLogs";
+import { ScreenName } from "~/const";
+import Collapsible from "~/newArch/components/Collapsible";
+import CopyButton from "~/newArch/components/CopyButton";
```

Comment:
You can replace ~/newArch by LLM/ if you want

Reply:
Good to know 👍